AsyncDisplayKit/Details/ASCollectionDataController.mm

```diff
@@ -88,8 +88,7 @@ - (void)prepareForInsertSections:(NSIndexSet *)sections
 
 - (void)willInsertSections:(NSIndexSet *)sections
 {
-  NSArray *keys = _pendingContexts.allKeys;
-  for (NSString *kind in keys) {
+  for (NSString *kind in _pendingContexts) {
     NSMutableArray<ASIndexedNodeContext *> *contexts = _pendingContexts[kind];
```

Comment:
I'm not pretty sure but can we actually use the block based enumeration api for NSDictionary here. As this would safe us a lookup.